kotlinx-coroutines-core/jvm/test/sync/IntervalLimiterTest.kt

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2016-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.coroutines.sync
+
+import kotlinx.coroutines.*
+import kotlinx.coroutines.test.*
+import org.junit.Test
```

Comment:
PS: I haven't checked the rest of Kotlin sources. Are they not using JUnit5 already?

Reply:
Nope xD I havn't written junit4 in, i think its 2 years now, so I have to hit the docs :-D
I even have a script to rewrite java-code from junit4 to 5, so I don't even have to think very much about junit4 anymore..